docs/query_attribute.rst

```diff
@@ -45,9 +45,9 @@ type
     Example: 'string'
 
 values
-    (Optional) In case the field has a list of possible values, they are listed here.
+    (Optional) In case the field has a list of possible values, they are listed here as a dictionary mapping the values to a definition (if available).
 
```

Comment:
maybe here add the remark you mentioned in an earlier commit: 

> We cannot make the distinction between {code: value} (vocubulary) and {value:definition} (values) reliably in XSD. The codes therefore need to replaced by values by the user. 

And maybe add the example of the mapping for e.g. Formele Stratigrafie under Example below
`df['lid1_lb'] = df['lid1'].map(fields['lid1']['values'])`

Reply:
I have added the example for 'values'. However it might be interesting to rework the full example below to (f.ex.) Formele Stratigrafie instead of Boring and move the mapping example there?

Reply:
Indeed, I was also looking at the Example. But it makes sense to focus the example on the WFS as it is an example for query attributes?
Maybe add it as an extra example of the 'values' list just before the discussing the OGC filter expressions?

Reply:
I moved it down to the example right before the OGC filter expressions.